scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -194,6 +194,38 @@ workflow Mutect2 {
     }
 
     Int m2_output_size = tumor_bam_size / scatter_count
+
+    if (run_ob_mm_filter) {
+        scatter (subintervals in SplitIntervals.interval_files ) {
+            call CollectF1R2Counts {
+                input:
+                    gatk_docker = gatk_docker,
+                    ref_fasta = ref_fasta,
+                    ref_fai = ref_fai,
+                    ref_dict = ref_dict,
+                    preemptible_attempts = preemptible_attempts,
+                    tumor_bam = tumor_bam,
+                    tumor_bai = tumor_bai,
+                    gatk_override = gatk_override,
+                    disk_space = tumor_bam_size + ref_size + disk_pad,
+                    intervals = subintervals,
+                    extra_intervals = ob_mm_filter_training_intervals,
+                    max_retries = max_retries
+            }
+        }
+
+        call LearnReadOrientationModel {
+            input:
+                alt_tables = CollectF1R2Counts.alt_table,
+                ref_histograms = CollectF1R2Counts.ref_histogram,
```

Comment:
ref has plural "histograms"  on LHS and singular "histogram" on RHS; alt has plural on both sides.

Reply:
fixed